nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Comment back end

# Description
Comment back end code
Partially resolves tickets #19, #20, #21, #22 

## Type of change

# Testing Instructions
- run code in ide's of your choice
- log in using account in database
- navigate to posts, post details, comments
- a partially formatted list of that post's comments should display

- testing additional functionality requires loading postman &  removing [Authorize] tag on CommentController
GET
- api/comment/{postId} will view a post's comments
- api/{postId}/{commentId} will pull a singular comment
Put & Post
- feed in comment that matches model
DELETE
- api/comment/{id}

# Checklist:
